ci/test_mac.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+#
+# Test JDBC for Mac
+#
+
+echo "DOWNLOADED"
+set -o pipefail
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $THIS_DIR/_init.sh
+source $THIS_DIR/scripts/set_git_info.sh
+
+
+export WORKSPACE=$GITHUB_WORKSPACE
+export SOURCE_ROOT=$GITHUB_WORKSPACE
+export SHOULD_GENERATE_COVERAGE_REPORT=1
```

Comment:
I don't we need this variable - we are not using it in the test_component.sh

Reply:
Removed.